pkgs/development/python-modules/mysql-connector/default.nix

```diff
@@ -13,6 +13,17 @@ in buildPythonPackage rec {
     sha256 = "1zb5wf65rnpbk0lw31i4piy0bq09hqa62gx7bh241zc5310zccc7";
   };
 
+  patches = [
+    # mysql-connector overrides MACOSX_DEPLOYMENT_TARGET to 11.
```

Comment:
Is there an upstream issue we could link? If not, one should probably be created.